cwms-data-api/src/main/java/cwms/cda/api/Controllers.java

```diff
@@ -121,15 +121,22 @@ public final class Controllers {
     public static final String TIMESERIES = "timeseries";
     public static final String LOCATIONS = "locations";
 
+    public static final String SPEC_ID = "spec-id";
+    public static final String LOCATION = "location";
```

Comment:
I'd go with "location-id", surprised there isn't an example already

Reply:
done

Reply:
you can remove this one?

Reply:
done